### PR TITLE
kth Closed

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: challenges/src/main/java/kth/LinkedList.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+
+    // Node class instantiate
+    static class Node {
+        int valueData;
+        Node next;
+
+        Node(int valueData) {
+            this.valueData = valueData;
+            this.next = next;
```
Comment (JamesDansie) [inline challenges/src/main/java/kth/LinkedList.java]: this is a bit odd. Either do this.next = null or pass in a next as an argument.
